### PR TITLE
Protect against matrix invert failure for ArgoNeuT

### PR DESCRIPTION
**Pull request by brballer**

### CODE REVIEW

File: ups/product_deps [hunk 1]
```diff
@@ -7,7 +7,7 @@ fcldir product_dir job
 fwdir  product_dir config_data
 #
 product         version
-larsim		v08_33_00
+larsim		v08_32_00
```
Comment (knoepfel) [inline ups/product_deps]: Changes to `ups/product_deps` should not be included in the PR; they will be included as part of the LArSoft release.

**Review by knoepfel — COMMENTED** [review]
Looks good, Bruce.  Can you update this PR to not include the ups/product_deps change?

**Comment by absolution1** [comment]
trigger build

**Comment by lgarren** [comment]
trigger build

**Comment by lgarren** [comment]
CI tests are normal (uboone and icarus are not up to date).

**Comment by ghost** [comment]
approve

**Comment by lgarren** [comment]
approve

**Comment by lgarren** [comment]
+1
